addons/crm/models/crm_stage.py

```diff
@@ -37,9 +37,17 @@ def default_get(self, fields):
     probability = fields.Float('Probability (%)', required=True, default=10.0, help="This percentage depicts the default/average probability of the Case for this stage to be a success")
     on_change = fields.Boolean('Change Probability Automatically', help="Setting this stage will change the probability automatically on the opportunity.")
     requirements = fields.Text('Requirements', help="Enter here the internal requirements for this stage (ex: Offer sent to customer). It will appear as a tooltip over the stage's name.")
-    team_id = fields.Many2one('crm.team', string='Team', ondelete='set null',
+    team_id = fields.Many2one('crm.team', string='Sales Channel', ondelete='set null',
         help='Specific team that uses this stage. Other teams will not be able to see or use this stage.')
     legend_priority = fields.Text('Priority Management Explanation', translate=True,
         help='Explanation text to help users using the star and priority mechanism on stages or issues that are in this stage.')
     fold = fields.Boolean('Folded in Pipeline',
         help='This stage is folded in the kanban view when there are no records in that stage to display.')
+
+    #This field for interface only
+    number_of_teams = fields.Integer('number_of_teams', compute='_compute_number_of_teams')
+
+    @api.multi
+    def _compute_number_of_teams(self):
+        for stage in self:
+            stage.number_of_teams = self.env['crm.team'].search_count([])
```

Comment:
Here you could perform the search_count outside of the loop. Otherwise when you display several teams you perform several times the same computation as it does not depend from the current record.